src/tests/suite/daos_obj.c

```diff
@@ -706,6 +706,7 @@ io_overwrite_large(void **state, daos_obj_id_t oid)
 	/* Disabled Pool Aggrgation */
 	rc = set_pool_reclaim_strategy(state, aggr_disabled);
 	assert_int_equal(rc, 0);
+	sleep(60);
```

Comment:
60 seconds looks too long to me, I think 10 seconds should be good enough. And could you add comments here to explain why we need sleep?

Reply:
Done